stl/inc/functional

```diff
@@ -1299,7 +1299,11 @@ inline void __stdcall _Function_deallocate_large_default_aligned(_Move_only_func
 
 template <size_t _Align>
 void __stdcall _Function_deallocate_large_overaligned(_Move_only_function_data& _Self) noexcept {
+#ifdef __cpp_aligned_new
     ::operator delete (_Self._Large_fn_ptr<void>(), align_val_t{_Align});
+#else // ^^^ __cpp_aligned_new / !__cpp_aligned_new vvv
+    _aligned_free(_Self._Large_fn_ptr<void>());

```

Comment:
These should be `_CSTD` qualified. Example:

https://github.com/microsoft/STL/blob/313aa593223e78b458ac50a5a46357601eebfe00/stl/inc/fstream#L405